docs/sources/shared/visualizations/legend-options-1.md

```diff
@@ -1,22 +1,29 @@
 ---
-title: Legend mode and legend placement
+title: Legend options
```

Comment:
should we add some internal comments to both of these legend option files to what visualizations they apply to / why there are two different legend option files? 😬

Reply:
Good call. Will do!